lookout/style/typos/generation.py

```diff
@@ -339,7 +347,7 @@ class DummyModel(Model):
         for key, val in self.checker._words.items():
             wordvals[delstrs_map[key]] = val
         tree["checker"]["_words"] = wordvals
-        tree["tokens"] = merge_strings(self.tokens)
+        tree["tokens"] = merge_strings(list(self.tokens))
```

Comment:
This should be deterministic:
```suggestion
        tree["tokens"] = merge_strings(sorted(self.tokens))
```